Makefile

```diff
@@ -56,23 +56,25 @@ MWCC_LD_VERSION := 1.1
 # Programs
 ifeq ($(WINDOWS),1)
   WINE :=
+  CPP     := $(DEVKITPPC)/bin/powerpc-eabi-cpp.exe -P
+  SHA1SUM := sha1sum
 else
   WINE ?= wine
   # Disable wine debug output for cleanliness
   export WINEDEBUG ?= -all
   # Default devkitPPC path
   DEVKITPPC ?= /opt/devkitpro/devkitPPC
+  CPP     := $(DEVKITPPC)/bin/powerpc-eabi-cpp -P
+  SHA1SUM := shasum
```

Comment:
Why was this changed? `sha1sum` is part of gnu coreutils so this should be the same on windows and linux

Reply:
I had to change this for Mac. Maybe we can just handle that in another if statement.

Reply:
Ah yeah I see, macos just has shasum. On linux shasum is part of perl, so I'd rather we keep using sha1sum on windows/linux and make this part mac-specific.